cms/templates/cms/toolbar/toolbar.html

```diff
@@ -4,7 +4,8 @@
      class="cms cms-reset
      {% if not user.is_authenticated %} cms-toolbar-auth{% endif %}
      {% if debug %} cms-toolbar-debug{% endif %}"
-     dir="ltr" data-touch-action="none">
+     lang="{{ LANGUAGE_CODE|default:"en-us" }}" dir="{{ LANGUAGE_BIDI|yesno:'rtl,ltr,auto' }}" 
+     data-touch-action="none">
```

Comment:
```suggestion
     data-touch-action="none">
```
Do you think we can try to make the full toolbar RTL compatible? You've come so far... 

In that case, we do not need to set any language or direction, but just take it from the page...